### ISSUE TITLE
Multi channels support

### ISSUE THREAD

**Issue by nado** [nado/go-b0tsec#3]
Ideally, with per channel configuration

**Comment by depado** [comment]
Do agree on that. I guess it shouldn't be too complicated to start a new bot inside a goroutine, so that none of the cocurrent bots collide with each other. Of course somme issues with the database might be predictable. Maybe create one DB per channel ?

**Comment by nado** [comment]
Yup, I thought about that, boltdb let us use boltdb objects as a key, so for the database it shouldnt be too complicated.
